xbmc/guilib/TextureFormats.h

```diff
@@ -0,0 +1,34 @@
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#define XB_FMT_MASK   0xffff ///< mask for format info - other flags are outside this
+#define XB_FMT_DXT_MASK   15
+#define XB_FMT_UNKNOWN     0
+#define XB_FMT_DXT1        1
+#define XB_FMT_DXT3        2
+#define XB_FMT_DXT5        4
+#define XB_FMT_DXT5_YCoCg  8
+#define XB_FMT_A8R8G8B8   16 // texture.xbt byte order (matches BGRA8)
+#define XB_FMT_A8         32
+#define XB_FMT_RGBA8      64
+#define XB_FMT_RGB8      128
+#define XB_FMT_OPAQUE  65536
```

Comment:
Also not the best to have a Kodi include on add-on headers, but to prevent compatibility faults and have the ___dev-kit___ includes complete is for me the better place there.

Another way (but a bit Balla-Balla) is to add a Translator and use a own header for add-on system. 
```cpp
int TextureFormatAddonToKodi(int a)
{
  int b = XB_FMT_UNKNOWN;
  if (a & ADDON_XB_FMT_UNKNOWN)
    b |= XB_FMT_UNKNOWN;
  ...
}
```
And the other way.

Background from me is to hold on end add-on's so much independent from Kodi as possible.